docs/3.0-migration-guide.d/remove_mbedtls_check_params_option.md

```diff
@@ -0,0 +1,49 @@
+Remove MBEDTLS_CHECK_PARAMS option
+----------------------------------
+
+This change affects the way of how parameters are validated.
+
+The option `MBEDTLS_CHECK_PARAMS` (disabled by default) enables certain kinds of
```

Comment:
That should be the past tense now: `enabled` - because it no longer does. Same for the next paragraphs.